### PR TITLE
Update dcos-history for TLS support

### PR DESCRIPTION
**Pull request by mellenburg**

### CODE REVIEW

**Comment by mellenburg** [comment]
@spahl @malnick this is ready to go

File: packages/dcos-history/extra/history/statebuffer.py [hunk 1]
```diff
@@ -7,10 +7,26 @@
...
 import requests
 
-FETCH_URL = "http://leader.mesos:5050/state-summary"
+logging.getLogger('requests.packages.urllib3').setLevel(logging.WARN)
+
 FETCH_PERIOD = 2
 FILE_EXT = '.state-summary.json'
 
+FETCH_URL = "http://leader.mesos:5050/state-summary"
+if 'STATE_SUMMARY_URI' in os.environ:
```
Comment (cmaloney) [inline packages/dcos-history/extra/history/statebuffer.py]: This doesn't make a lot of sense as a separate / different STATE_SUMMARY_URI. Can we make it have one name throught? Either always FETCH_URL or always STATE_SUMMARY_URL?

**Comment by cmaloney** [comment]
One thing, but otherwise looking good

**Comment by cmaloney** [comment]
:ship:

**Pull request merged**